src/Symfony/Component/Messenger/Tests/Middleware/DispatchAfterCurrentBusMiddlewareTest.php

```diff
@@ -301,11 +301,11 @@ private function expectHandledMessage($message): Callback
         });
     }
 
-    private function willHandleMessage(): ReturnCallback
+    private function handleMessage(): \Closure
```

Comment:
I would name it `handleMessageCallback()` as this method does not handle a message.

Reply:
Agree :+1: